docs/en/query_language/functions/rounding_functions.md

```diff
@@ -78,6 +78,92 @@ round(3.55, 1) = 3.6
 round(3.65, 1) = 3.6
 ```
 
+**See Also** 
+
+- [roundBankers](#roundbankers)
+
+## roundBankers(x[, N]) {#roundbankers}
+
+Banker's rounding is a method of rounding fractional numbers, which differs from the usual arithmetic rounding in cases when the rounding number is a half between two numbers, it is rounded to the nearest even number. E.g. 3.5 rounds up to 4, 2.5 rounds down to 2.
+
+Banker's rounding allows to sum rounded numbers without biasing a result.
+
+Examples of rounding 1.5, 2.5, 3.5 and 4.5:
+
+The sum of the original numbers: 1.5 + 2.5 + 3.5 + 4.5 = 12
+
+The sum of banker's rounded numbers: 2 + 2 + 4 + 4 = 12
+
+The sum of arithmetic rounded numbers: 2 + 3 + 4 + 5 = 14.
+
+
+**Syntax** 
+
+```sql
+roundBankers(expression [, decimal_places])
+```
+
+**Parameters** 
+
+- `expression` — A number to be rounded. Can be any [expression](../syntax.md#syntax-expressions) returning the numeric [data type](../../data_types/index.md#data_types). E.g.
+- `decimal-places` — An integer value.
```

Comment:
Don't use the future time in the following list. We write documentation in the present simple time, excepting very rare cases.